lunch/test/test_common.py

```diff
@@ -114,7 +114,7 @@ def test_add_commands_with_dependencies(self):
 
         def _cl1():
             all = self._master.get_all_commands()
-            self.failUnlessEqual(len(all), 3)
+            self.assertEqual(len(all), 3)
```

Comment:
We should stay with failUnlessEqual, here, since this is a unit test. 
(though, this is not required, since both work)